pages/user/project.js

```diff
@@ -0,0 +1,163 @@
+                <h5>About Organization</h5>
+              </div>
+              <hr className="hr" />
+              <div className="row d">
+                <p>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+                   Ut laoreet, neque in consectetur imperdiet,
+                  lacus sem lobortis enim, vitae accumsan ligula nulla ac lacus.
+                   Pellentesque in augue eget nuncdictum
+                  sodales. In blandit pellentesque vestibulum.
+                   Interdum et malesuada fames ac ante ipsum primis in
+                  faucibus. In vel sollicitudin leo.
+                </p>
+              </div>
+              <div className="row comment">
+                <p>12 Comments</p>
+              </div>
+              <div className="row comments">
+                <div className="row cmt">
+                  <div className="column column-20">
+                    <img src="/assets/images/avatar-1.png" alt="organisation-pic" />
+                  </div>
+                  <div className="column column-80">
+                    <p>
+                      Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+                       Rem a sit excepturi quisquam ullam? Quia
+                      inventore consectetur debitis, error voluptas
+                       officiis dolores excepturi consequuntur officia.
+                    </p>
+                    <div className="row post-time">
+                      <em>posted 5h ago</em>
+                    </div>
+                  </div>
+                </div>
+                <div className="row cmt">
+                  <div className="column column-20">
+                    <img src="/assets/images/avatar-1.png" alt="organisation-pic" />
+                  </div>
+                  <div className="column column-80">
+                    <p>
+                      Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+                       Rem a sit excepturi quisquam ullam? Quia
+                      inventore consectetur debitis, error voluptas officiis
+                       dolores excepturi consequuntur officia.
+                    </p>
+                    <div className="row post-time">
+                      <em>posted 5h ago</em>
+                    </div>
+                  </div>
+                </div>
+                <div className="pos">
+                  <div className="row cmt post">
+                    <div className="column column-10">
+                      <img src="/assets/images/avatar-1.png" alt="organisation-pic" />
+                    </div>
+                    <div className="column column-67">
+                      <input type="text" name="write-comment" id="write-comment" placeholder="Write a comment" />
+                    </div>
+                    <div className="column column-10">
+                      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
```

Comment:
## Nitpick
We can make use of `font-awesome` library instead of grabbing the svg file itself.